services/ui-src/src/components/forms/AdminBannerForm.test.tsx

```diff
@@ -64,11 +73,24 @@ describe("Test AdminBannerForm component", () => {
       ),
     });
   });
+
+  test("Calls errorHandler if writeBanner throws error", async () => {
+    const result = render(
+      adminBannerFormComponent(mockWriteAdminBannerWithError)
+    );
+    const form = result.container;
+    await fillOutForm(form);
+    const submitButton = screen.getByRole("button");
+    await userEvent.click(submitButton);
+    await expect(errorHandler).toHaveBeenCalled();
+  });
```

Comment:
Another pattern you'll see a lot. Add test to check that the `errorHandler` is called if an error upstream is thrown.